src/TTImp/ProcessType.idr

```diff
@@ -267,7 +267,7 @@ processType {vars} eopts nest env fc rig vis opts (MkImpTy tfc nameFC n_in ty_ra
          addNameLoc nameFC n
 
          log "declare.type" 1 $ "Processing " ++ show n
-         log "declare.type" 5 $ "Checking type decl " ++ show n ++ " : " ++ show ty_raw
+         log "declare.type" 5 $ "Checking type decl " ++ show n ++ " : " ++ show rig ++ " " ++ show ty_raw
```

Comment:
This will generate logs that look more like an actual definition:

```suggestion
         log "declare.type" 5 $
           unwords ["Checking type decl:", show rig, show n, ":", show ty_raw]
```